### ISSUE TITLE
Issue with "last_session_files" ? GTtuner gets slow when switching panes.

### ISSUE THREAD

**Issue by Scachi** [J2Kbr/GtunerIV#370]
After using GTuner IV a few days it suddenly gets very slow when swichting panes between "Device Monitor" and "GPC Script IDE".

Everytime I delete the 'last_session_files' line in the gtuner.cfg file and restart gtuner it works fine without that annoying delay when switching panes again for some time.
```
last_session_files=@Variant(\0\0\0\b\0\0\0\x1\0\0\0\xb2\0\x30\0\x30\0\0\0\x43\0:\0/\0U\0s\0\x65\0r\0s\0/\0s\0\x63\0\x61\0\x63\0h\0.\0\x44\0\x45\0S\0K\0T\0O\0P\0-\0P\0O\0\x30\0J\0R\0V\0\x38\0/\0\x44\0o\0\x63\0u\0m\0\x65\0n\0t\0s\0/\0T\0\x32\0/\0_\0_\0_\0T\0\x32\0_\0M\0y\0_\0S\0t\0u\0\x66\0\x66\0/\0P\0U\0\x42\0G\0_\0v\0\x32\0/\0z\0_\0_\0\x64\0\x65\0\x66\0\x61\0u\0l\0t\0_\0x\0\x62\0o\0x\0.\0g\0i\0t\0\0\0\x2\0\0\0\0)
```

**Comment by Scachi** [comment]
It isn't limited to switching between "Device Monitor" and "GPC SCript IDE"..that delay happens on any panel switching. Same for "Online Resource" -> "Input Translator" and so on.

**Comment by J2Kbr** [comment]
I couldn't reproduce in my computer. Do you usually keep several files opened? If yes, please let me know how many so I can try reproduce. Also, this was noticed only with the latest update (1.07), or was also happening with older versions? Thanks.

**Comment by Scachi** [comment]
I have 3 to 5 files open most of the time as I tend to split my scripts into several .gph files like for Interactive Configuration, online resource, the script itself, additional header files.

That delay ranges from 0.5sec to 1.5sec when switching panels.

Not 100% but I think I first encountered this issue somewhere in the beta period of computer vision.
Is it possible that it belongs to this one : "* Session: The order of files opened in the GPC Script IDE are now correctly restored" ?
Would that make sense as deleting the "last_session_files=" line in the ini files removes the delay ?

I don't have any older gtuner version around anymore to test with different versions.

**Comment by Scachi** [comment]
Scratch the information above. It looks like the slow down depends only on having an Input Translator file opened.
Here is an video navigating the panels without any opens files first, then I open a .gpc file and it still works fast. Then closing the .gpc file and opening the .git file it gets really slow.
I click on the destination panel it and takes a few moments for the panel to open then.
Video Link:
https://streamable.com/14qg5p

Without an .git/IT file opened it switches the panels instantly.
It looks like it is rendering/accessing the IT files data each time I do switch panels, no matter what the source or destination panel is.
I would understand that it impacts when opening the IT panel, but not when switching the other ones.
As I have noticed this one recently only it may be me not using  .git files much, so I can't really tell when this has started or if it is this way for a long time already.

Can this be improved ?

**Comment by J2Kbr** [comment]
Thank you for the information. Issue found and fixed (1.08, to be released soon). The problem was with the combo-boxes, specially the ones to select the keyboard key. Optimizations were made to improve performance.

**Comment by Scachi** [comment]
Works perfectly now. No more performance issues when switching panel. Thank you for the quick fix
